From 6a4f980921195c55752da50e8d6f160ba730f398 Mon Sep 17 00:00:00 2001
From: Victor Lowther <victor.lowther@gmail.com>
Date: Wed, 2 Jun 2010 07:07:55 -0500
Subject: [PATCH 136/197] Just look for cryptroot instead of /sbin/cryptroot

We know what the path is and what it contains in the initrd, and
not everyone puts cryptroot in /sbin
---
 modules.d/90crypt/cryptroot-ask.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.d/90crypt/cryptroot-ask.sh b/modules.d/90crypt/cryptroot-ask.sh
index 13487c8..2f39667 100755
--- a/modules.d/90crypt/cryptroot-ask.sh
+++ b/modules.d/90crypt/cryptroot-ask.sh
@@ -68,7 +68,7 @@ if [ $ask -gt 0 ]; then
     # flock against other interactive activities
     { flock -s 9; 
 	echo -n "$device ($luksname) is password protected"
-	/sbin/cryptsetup luksOpen -T1 $1 $luksname 
+	cryptsetup luksOpen -T1 $1 $luksname 
     } 9>/.console.lock
 fi
 
-- 
1.7.0.1

